cfme/markers/env_markers/provider.py

```diff
@@ -147,7 +149,7 @@ def data_provider_types(provider):
     return all_types()[provider.type_name]
 
 
-@attr.s
+@attr.s()
```

Comment:
Q: any special reason Mike? we are not passing any arg.

Reply:
unintentional, good find!